File: app/src/main/java/com/mrchandler/disableprox/SensorDisablerPreferenceProvider.java
package com.mrchandler.disableprox;

import com.crossbowffs.remotepreferences.RemotePreferenceProvider;
import com.mrchandler.disableprox.util.Constants;

/**
 * @author Wardell Bagby
 */

public class SensorDisablerPreferenceProvider extends RemotePreferenceProvider {
    public SensorDisablerPreferenceProvider() {
        super("com.mrchandler.disableprox", new String[]{Constants.PREFS_FILE_NAME});
    }
}

File: app/src/main/java/com/wardellbagby/sensordisabler/SensorDisablerPreferenceProvider.kt
package com.wardellbagby.sensordisabler;

import com.crossbowffs.remotepreferences.RemotePreferenceProvider;
import com.wardellbagby.sensordisabler.util.Constants;

public class SensorDisablerPreferenceProvider extends RemotePreferenceProvider {
  public SensorDisablerPreferenceProvider() {
    super("com.wardellbagby.sensordisabler", new String[] { Constants.PREFS_FILE_NAME });
  }
}
